.github/workflows/stale.yml

```diff
@@ -0,0 +1,95 @@
+name: Stale
+
+on:
+  workflow_dispatch:
+    inputs:
+      dryrun:
+        description: "dryrun: Preview stale issues/prs without marking them (true|false)"
+        required: true
+        default: "true"
+  schedule:
+    - cron: 0 4 * * *
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - id: stale_issue
+        uses: KJ002/read-yaml@1.5
+        with:
+          file: .github/messages.yml
+          key-path: stale-issue
+      - id: close_issue
+        uses: KJ002/read-yaml@1.5
+        with:
+          file: .github/messages.yml
+          key-path: close-issue
+      - id: stale_pr
+        uses: KJ002/read-yaml@1.5
+        with:
+          file: .github/messages.yml
+          key-path: stale-pr
+      - id: close_pr
+        uses: KJ002/read-yaml@1.5
+        with:
+          file: .github/messages.yml
+          key-path: close-pr
```

Comment:
I'm a bit uneasy to use a 3rd party action for this use case since the code is run in every repo. Could we do the same with [actions/github-script](https://github.com/actions/github-script) and the various libaries mentioned there?

Porting https://github.com/KJ002/read-yaml/blob/main/src/index.ts into a composite action in conda/actions would be not a huge amount of effort I think. What do you think?